src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cookies.cs

```diff
@@ -262,6 +266,7 @@ public async Task GetAsync_SetCookieContainerAndMultipleCookieHeaders_BothCookie
                     HttpRequestData requestData = await serverTask;
                     string cookieHeaderValue = GetCookieValue(requestData);
 
+#if NETFRAMEWORK
                     // Multiple Cookie header values are treated as any other header values and are
```

Comment:
I'm leaning towards simplifying this to:
```C#
#if NETFRAMEWORK
var separators = new string[] { "; ", ", " };
#else
var separators = new string[] { "; " };
#endif
var cookieValues = cookieHeaderValue.Split(separators, StringSplitOptions.None);
...
```
We can keep the comment about cookie container and how the mixed separators got in.
What do you think?

Reply:
Sure, fixed.